CareKitStore/CareKitStoreTests/Structs/TestOutcomeValue.swift

```diff
@@ -178,12 +178,15 @@ class TestOutcomeValue: XCTestCase {
 
     func testEqualityOfEncodings(outcome1: OCKOutcomeValue, outcome2: OCKOutcomeValue) {
         let encoder = JSONEncoder()
+        encoder.outputFormatting = .sortedKeys
```

Comment:
I suspect the original problem (https://github.com/carekit-apple/CareKit/pull/712#discussion_r1509756329) with comparing `Data` types was here because the keys weren't sorted. Basically the `Data` was the same size, but the encoder encoded the key/values in different places, resulting in different `Data`.